ouroboros-consensus/src/Ouroboros/Storage/ChainDB/Mock.hs

```diff
@@ -45,6 +44,9 @@ openDB cfg initLedger blockHeader = do
         query' :: (Model blk -> a) -> m a
         query' = atomically . query
 
+        queryE' :: (Model blk -> Either (ChainDbError blk) a) -> m a
```

Comment:
These names aren't especially helpful. Given the worries about not many people knowing the chainDB code, it might be nicer to have more explanatory names here.

Reply:
The `BlockGCed` error is part of `BlockFetchServerException`, that's why I don't use it. But maybe I should rename it and use it here too.

Reply:
I'm not that worried about `queryE'` and so on being unhelpful. These are all local helpers with self-explanatory type signatures. If these names were longer, then `ChainDB` record implementation  would become less readable (too long lines), and this is the most important part.